### PR TITLE
Promoting changes to main

### PR DESCRIPTION
**Pull request by piyush-bitpack**

### CODE REVIEW

File: src/pages/dapp/index.tsx [hunk 1]
```diff
@@ -313,7 +358,8 @@ function ReviewDialog(props) {
           </svg>
         </button>
         <Column>
-          Have you opened or downloaded dapp before posting review?
+          <p>You have not open or download dapp before posting review.</p>
```
Comment (thebitparticle) [inline src/pages/dapp/index.tsx]: you have not `opened`

(or) 

You did not visit or download the dapp before to post a review

File: src/pages/dapp/index.tsx [hunk 2]
```diff
@@ -459,13 +505,13 @@ function AppRatingList(props) {
           </Link>
         </Row>
       )}
-      <div className="absolute inset-0">
+      {/* <div className="absolute inset-0">
         <div className="backdrop-filter backdrop-blur-sm absolute w-full h-full flex items-center justify-center ">
```
Comment (thebitparticle) [inline src/pages/dapp/index.tsx:509]: this can be removed now right?

Comment (thebitparticle) [inline src/pages/dapp/index.tsx]: this commented block can be removed now right?

**Review by thebitparticle — CHANGES_REQUESTED** [review]
check the comments please

**Review by r4881t — APPROVED** [review]

File: src/pages/dapp/index.tsx (hunk 1 above)
Comment (piyush-bitpack) [reply src/pages/dapp/index.tsx]: You have not opened or downloaded Dapp before posting a review.
this will be good.

**Pull request merged**